docs/specifications/tests/DNSSEC-TP/dnssec02.md

```diff
@@ -110,6 +112,7 @@ BROKEN_DS                     | ERROR
 BROKEN_RRSIG                  | ERROR
 DNSKEY_KSK_NOT_SEP            | NOTICE
 DNSKEY_NOT_ZONE_SIGN          | ERROR
+DS02_ALGO_NOT_SUPPORTED_BY_ZM | NOTICE
```

Comment:
No, there is no place where this is described, and I have been waiting for some questions and comments on the new format. :-)

The idea was to use one letter when possible and two when needed, but I am open for

1. Always having two letters.
2. Creating a document on the prefix (a good idea).

The idea is not that the prefix should be magic in the code. This should be for human consumption just as "ALGO" and "KSK" are not magic and just to make it easier for the human brain to read and remember.

My Idea was to change the tag when there is a major change of the specification, as you could see that I did not consider this PR to be. Do you think it should be done otherwise?

Below is how I thought the prefix (for test case "01") should be and a proposal with the change you ask for.

Module           | Short code | Always two letters
:----------------|:-----------|:------------
Address          | A01        | AD01 
Basic            | B01        | BA01
Connectivity     | CN01       | CN01
Consistency      | CS01       | CS01
DNSSEC           | DS01       | DS01
Delegation       | DE01       | DE01
Nameserver       | N01        | NS01
Syntax           | S01        | SY01
Zone             | Z01        | ZO01